lib/interface/cli/commands/trigger/event/get.cmd.js

```diff
@@ -0,0 +1,42 @@
+const debug = require('debug')('codefresh:cli:get:trigger-event');
+const Command = require('../../../Command');
+const _ = require('lodash');
+const { trigger } = require('../../../../../logic').api;
+const { specifyOutputForSingle, specifyOutputForArray } = require('../../../helpers/get');
+const getRoot = require('../../root/get.cmd');
+
+const command = new Command({
+    command: 'trigger-event <event-uri>',
```

Comment:
should get all triggers-events or a specific trigger-event by ID/NAME.

event-uri should be a possible filter

Reply:
event-URI is the ID for trigger-event

Reply:
event-uri is the outcome of creating a trigger-event as i understood.
it should be possible for a user to provide the unique name of the created trigger-event.

if you don't want to also have an id it is fine but the get/delete operations should work with the passed unique name the user will pass.